assets/sass/_form.scss

```diff
@@ -1,6 +1,6 @@
 #mkto-form {
   margin-top: 20px;
-  width: 70%;
+  width: 100%;
```

Comment:
I'm not familiar with the CSS for KT, so I'm going to assume changes here and below are good.

Reply:
Thanks for taking a look, @bbejeck. The form styling changes were necessary to handle the narrower subscribe card. We will have another chance verify they render correctly when we stage the site for release. (Separate PR with staging link coming shortly.)